### PR TITLE
Expose PluginStore to Python

### PR DESCRIPTION
**Pull request by jdent02**

### CODE REVIEW

File: src/appleseed/renderer/modeling/project/project.h [hunk 1]
```diff
@@ -48,6 +48,7 @@
 #include "renderer/modeling/surfaceshader/surfaceshadertraits.h"
 #include "renderer/modeling/texture/texturetraits.h"
 #include "renderer/modeling/volume/volumetraits.h"
+#include "renderer/utility/pluginstore.h"
```
Comment (dictoon) [inline src/appleseed/renderer/modeling/project/project.h]: Remove. `PluginStore` is foward-declared on line 83.

File: src/appleseed.studio/CMakeLists.txt [hunk 2]
```diff
@@ -433,6 +433,7 @@ set (appleseedpython_sources
     ../appleseed.python/bindmurmurhash.cpp
     ../appleseed.python/bindobject.cpp
     ../appleseed.python/bindpostprocessingstage.cpp
+    ../appleseed.python/bindpluginstore.cpp
```
Comment (dictoon) [inline src/appleseed.studio/CMakeLists.txt]: Sort alphabetically.

File: src/appleseed.python/bindpluginstore.cpp [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+
+// appleseed.renderer headers.
+#include "renderer/api/project.h"
```
Comment (dictoon) [inline src/appleseed.python/bindpluginstore.cpp]: `renderer/api/project.h` needs to be updated to include `pluginstore.h`.

**Comment by jdent02** [comment]
Fixed.

**Pull request closed without merging**